doc2docx.py

```
@@ -1,221 +1,222 @@
+    finally:
+        # Ensure Word is closed
+        if word_app:
+            try:
+                word_app.Quit()
+            except Exception:
+                pass
+        
+        progress.finish()
+    
+    # Summary
+    logging.info(f"\nConversion complete!")
```

Ruff (0.15.1)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)